@kiva/kv-components/vue/KvVotingCard.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+	<div
+		class="
+		kv-voting-card
+		tw-bg-white
+		tw-rounded
+		tw-shadow
+		tw-p-1
+		tw-flex
+		tw-flex-col
+		tw-items-center
+		tw-justify-between
+		tw-max-w-300"
+	>
+		<div class="tw-relative tw-flex tw-w-full tw-bg-black tw-rounded tw-mb-1">
+			<kv-borrower-image
+				class="tw-rounded"
+				:alt="`Photo of ${borrowerName}`"
+				:aspect-ratio="aspectRatio"
+				:default-image="defaultImage"
+				:hash="hash"
+				:images="images"
+				:photo-path="photoPath"
+			/>
+			<p
+				class="
+				tw-absolute
+				tw-bottom-1
+				tw-left-1
+				tw-text-primary
+				tw-bg-white
+				tw-rounded
+				tw-text-h4
+				tw-inline-flex
+				tw-items-center"
+				style="padding: 2px 6px; text-transform: capitalize;"
+			>
+				<kv-material-icon
+					class="tw-h-2 tw-w-2"
+					:icon="mapMarkerIcon"
+				/>
+				{{ borrowerName }}, {{ country }}
```

Comment:
Do you think we'll need to truncate or remove the country in some cases do to being to long? I think there's a tw- class for that but don't recall exactly... It might be useful to create a story variation that tests what this looks like when the combination of the name and country spill out of the card width...